Index: .env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.env b/.env
new file mode 100644
--- /dev/null	(date 1733653971215)
+++ b/.env	(date 1733653971215)
@@ -0,0 +1,6 @@
+SECRET_KEY=django-insecure-=r8mz08*arh(6+uwtm6tyuc_6im%pi8o3$_d-7+33nwjt#q@mu
+DB_NAME=hikingtrailsuserprob
+DB_USER=postgres
+DB_PASS=admin
+DB_HOST=127.0.0.1
+DB_PORT=5432
Index: .idea/dataSources/8c39438b-ddae-424b-85da-792d2a32ea72/storage_v2/_src_/database/hikingtrailsuserprob.5VkV7A/schema/public.abK9xQ.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#n:public\r\n!<md> [7524, 0, null, null, -2147483648, -2147483648]\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/dataSources/8c39438b-ddae-424b-85da-792d2a32ea72/storage_v2/_src_/database/hikingtrailsuserprob.5VkV7A/schema/public.abK9xQ.meta b/.idea/dataSources/8c39438b-ddae-424b-85da-792d2a32ea72/storage_v2/_src_/database/hikingtrailsuserprob.5VkV7A/schema/public.abK9xQ.meta
--- a/.idea/dataSources/8c39438b-ddae-424b-85da-792d2a32ea72/storage_v2/_src_/database/hikingtrailsuserprob.5VkV7A/schema/public.abK9xQ.meta	(revision 00c7565a73ef27273fb84f8625d2009a8b78a1a0)
+++ b/.idea/dataSources/8c39438b-ddae-424b-85da-792d2a32ea72/storage_v2/_src_/database/hikingtrailsuserprob.5VkV7A/schema/public.abK9xQ.meta	(date 1733653027005)
@@ -1,2 +1,2 @@
 #n:public
-!<md> [7524, 0, null, null, -2147483648, -2147483648]
+!<md> [7557, 0, null, null, -2147483648, -2147483648]
